src/components/card/CardSidebarTabDetails.vue

```diff
@@ -93,21 +106,31 @@
 				<span class="hidden-visually">{{ t('deck', 'Due date') }}</span>
 			</div>
 			<div class="section-details">
-				<NcDatetimePicker v-model="duedate"
+				<NcDatetimePicker v-if="(!copiedCard.duedate && !card.done) || copiedCard.duedate"
+					v-model="duedate"
 					:placeholder="t('deck', 'Set a due date')"
 					type="datetime"
 					:minute-step="5"
 					:show-second="false"
 					:lang="lang"
 					:formatter="format"
-					:disabled="saving || !canEdit"
+					:disabled="saving || !canEdit || card.done"
 					:shortcuts="shortcuts"
 					confirm />
 				<NcActions v-if="canEdit">
-					<NcActionButton v-if="copiedCard.duedate" icon="icon-delete" @click="removeDue()">
+					<NcActionButton v-if="copiedCard.duedate && !card.done" icon="icon-delete" @click="removeDue()">
 						{{ t('deck', 'Remove due date') }}
 					</NcActionButton>
 				</NcActions>
+
+				<div class="done">
+					<div v-if="!copiedCard.duedate && card.done" class="no-due">
+						{{ t('deck', 'No due date') }}
```

Comment:
I think this is a bit weird, UX wise. I would expect this to show "Done", even if no due date was set. Or "Done without due date" cc @nextcloud/designers

Reply:
My bad, I can't read. 
![image](https://user-images.githubusercontent.com/986878/201154026-9d3a52c6-cc76-4aed-9f3b-c4ac7b1d4809.png)